packages/odie-client/src/components/message/custom-a-link.tsx

```diff
@@ -41,7 +43,7 @@ const CustomALink = ( {
 			<a
 				className="odie-sources-link"
 				href={ transformedHref }
-				target="_self"
```

Comment:
We should fallback to  `_self` if `target` doesn't exist.

Reply:
It is, the default value for it is _self, check [here](https://github.com/Automattic/wp-calypso/pull/96068/files#diff-56b82e97e9256975f119fd158e7e9a5d37824c638c1168daae01d72b820a2044R16)